controllers/drplacementcontrol_controller.go

```diff
@@ -635,6 +645,20 @@ func (r *DRPlacementControlReconciler) Reconcile(ctx context.Context, req ctrl.R
 	return r.reconcileDRPCInstance(d, logger)
 }
 
+func (r *DRPlacementControlReconciler) setProgressionAndUpdate(
+	ctx context.Context, drpc *rmn.DRPlacementControl,
+) (ctrl.Result, error) {
+	updated := updateDRPCProgression(drpc, rmn.ProgressionSetDeleting, r.Log)
+
+	if updated {
+		if err := r.Update(ctx, drpc); err != nil {
```

Comment:
You want to make sure that the status is updated right?

Reply:
ok. corrected it now.